packages/css-engine/src/core/rules.ts

```diff
@@ -142,9 +142,20 @@ export class FontFaceRule {
     this.options = options;
   }
   get cssText() {
-    const { fontFamily, fontStyle, fontWeight, fontDisplay, src } =
-      this.options;
-    return `@font-face {\n  font-family: ${fontFamily}; font-style: ${fontStyle}; font-weight: ${fontWeight}; font-display: ${fontDisplay}; src: ${src};\n}`;
+    const decls = [];
+    let property: keyof FontFaceOptions;
+    for (property in this.options) {
```

Comment:
Each option can be handled explicitly without loop

Reply:
yep, turned out same LOC